src/frontend/src/lib/components/ui/ButtonCloseModal.svelte

```diff
@@ -2,11 +2,8 @@
 	import Button from '$lib/components/ui/Button.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { modalStore } from '$lib/stores/modal.store';
-	import type { ButtonColorStyle } from '$lib/types/style';
-
-	export let colorStyle: ButtonColorStyle = 'secondary';
 </script>
 
-<Button paddingSmall {colorStyle} type="button" fullWidth on:click={modalStore.close}>
+<Button paddingSmall type="button" fullWidth on:click={modalStore.close}>
```

Comment:
I think here you should do `colorStyle="secondary"` because `Button` defaults to `primary`